CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). (Patch version X.Y.0 is implied if not specified.)
 
 ## [Unreleased](https://github.com/USGS/time-series-services/compare/time-series-services/compare/0.4.0...master)
+### Changed
```

Comment:
I'm not sure we need a changelog entry representing our internal operations.

Reply:
I concur, these will get out-of-hand if we list more than changes visible to the consumer of the api.